.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Owning team for this repo
+* @dbt-labs/cloud-cli
```

Mind using `@dbt-labs/cloud-clients-foundations`? Cloud-cli would work, though foundations is the group we're using going forward

```suggestion
* @dbt-labs/cloud-clients-foundations
```